repose-aggregator/components/filters/header-user-filter/pom.xml

```diff
@@ -48,6 +48,7 @@
         <dependency>
             <groupId>org.scalatest</groupId>
             <artifactId>scalatest_${scala.MajDotMin}</artifactId>
+            <scope>test</scope>
```

Comment:
I had to verify this one. Making the scope explicit is legit, but it looks like it was being dependency managed anyway.

```
components/filters/header-user-filter  repose-8-staging ✔                                                                                  
▶ mvn dependency:tree -Dscope=compile
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Building Repose Components - Client Identity By Header 8.0.0.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- maven-dependency-plugin:2.8:tree (default-cli) @ header-user-filter ---
[INFO] org.openrepose:header-user-filter:jar:8.0.0.0-SNAPSHOT
[INFO] +- org.openrepose:repose-core-api:jar:8.0.0.0-SNAPSHOT:compile
[INFO] |  +- com.google.guava:guava:jar:18.0:compile
[INFO] |  \- com.yammer.metrics:metrics-core:jar:2.2.0:compile
[INFO] +- org.jvnet.jaxb2_commons:jaxb2-basics-runtime:jar:0.6.3:compile
[INFO] +- org.openrepose:commons-utilities:jar:8.0.0.0-SNAPSHOT:compile
[INFO] |  +- org.openrepose:http-client-service-api:jar:8.0.0.0-SNAPSHOT:compile
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.4.1:compile
[INFO] |  |  \- org.apache.httpcomponents:httpcore:jar:4.4.1:compile
[INFO] |  +- commons-pool:commons-pool:jar:1.6:compile
[INFO] |  +- org.apache.commons:commons-lang3:jar:3.3.2:compile
[INFO] |  +- com.fasterxml.jackson.core:jackson-core:jar:2.4.0:compile
[INFO] |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.4.0:compile
[INFO] |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.4.0:compile
[INFO] |  +- org.apache.logging.log4j:log4j-api:jar:2.3:compile
[INFO] |  \- commons-codec:commons-codec:jar:1.10:compile
[INFO] +- org.scala-lang:scala-library:jar:2.11.7:compile
[INFO] +- com.typesafe.scala-logging:scala-logging-slf4j_2.11:jar:2.1.2:compile
[INFO] |  \- com.typesafe.scala-logging:scala-logging-api_2.11:jar:2.1.2:compile
[INFO] +- org.slf4j:slf4j-api:jar:1.7.7:compile
[INFO] +- javax:javaee-web-api:jar:6.0:provided
[INFO] \- javax.transaction:transaction-api:jar:1.1:provided
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 0.905 s
[INFO] Finished at: 2016-04-13T10:15:13-05:00
[INFO] Final Memory: 15M/309M
[INFO] ------------------------------------------------------------------------
```